src/useAnimateGroup.ts

```diff
@@ -62,16 +62,14 @@ export default function useAnimateGroup(props: Props): {
       styleTagRef.current[i].reverse = result.styleTag;
     });
 
+    const styleTags = [...styleTagRef.current];
+    const animationNames = [...animationNamesRef.current];
```

Comment:
can we skip those variables? save some bytes.

Reply:
I can not directly use `styleTagRef.current` or `animationNamesRef.current` in the cleanup function, but making a copy of the arrays could be skipped. since we can just store the refs' references